contracts/colony/Colony.sol

```diff
@@ -191,6 +192,12 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     require(msg.sender == colonyNetworkAddress, "colony-access-denied-only-network-allowed");
     // Function only valid on the Meta Colony
     require(address(this) == IColonyNetwork(colonyNetworkAddress).getMetaColony(), "colony-access-denied-only-meta-colony-allowed");
+    // Not callable on Xdai
+    uint256 chainId = getChainId();
+    if (chainId == 100 || chainId == 265669100) {
```

Comment:
I'd rather these numbers be defined as constants somewhere, probably in the `GetChainId` contract.